core/src/main/java/com/example/AutomatedReviewTest.java

```diff
@@ -0,0 +1,36 @@
+package com.example;
+
+import java.util.*;
+
+// The goal of this code is to find out if code analysis will automatically add
+// commentary to a pull request when it finds problems with the code.
+
+public abstract class AutomatedReviewTest extends Object {
+
+  public boolean hello(Optional unusedRaw)
+  {
+    float x = 3.0f;
+    if (x + 2.0f * 1.0f == 5.0f)
+      System.out.println("Yay\u0021".toUpperCase());
+      
+    return true;
+  }
+
+  private AutomatedReviewTest() { hello(Optional.of(this)); }
+  
+  public static void main(String... args) throws SomeError {
+  	}
+}
+
+final class SomeError extends Error {
+
+	public static java.util.Collection<Object> collection = null;
+	
+	protected Collection getCollectionLazily() {
+		if (collection == null) {
+			collection = new ArrayList<Object>();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make the enclosing method "static" or remove this set. ([squid:S2696](https://sonarcloud.io/organizations/coderanch-corral/rules#rule_key=squid%3AS2696))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.coderanch%3Ablackjack&pullRequest=9&issues=AWY_zChlvOM6zLh9Plcp&open=AWY_zChlvOM6zLh9Plcp)

[ISSUE_KEY]: AWY_zChlvOM6zLh9Plcp